keepassxc-browser/_locales/en/messages.json

```diff
@@ -132,7 +136,7 @@
         "description": "Password icon title text."
     },
     "passwordGeneratorError": {
-        "message": "Cannot receive generated password.<br />Is KeePassXC running?",
+        "message": "Cannot receive generated password.\nIs KeePassXC running?",
```

Comment:
This line break is not actually displayed. A way to display it is styling the div with `white-space: pre-wrap`, but I'm not sure if there are more elegant solutions.

Reply:
This message is also shown if KeePassXC is running, but not connected. Should the message be changed to "Is KeePassXC running and connected?"

Reply:
Line changes in translations are tricky. It would be the best to avoid them. And yes, this message should be changed.

Reply:
Resolved with https://github.com/keepassxreboot/keepassxc-browser/pull/128/commits/ec3eb66dbf5859363bd8efddbbc288641b70f92e.